moc-health-check/health_checker.py

```diff
@@ -0,0 +1,144 @@
+    command = open_subprocess(cmd)
+    print(cmd)  	
+    output = command.stdout.read()                                                     
+    output = str(output, "utf-8")
+    if '"status": true' in output:
+        success_pfioh_push = success_pfioh_push + 1
+        cmd = 'bash %s/scripts/run_pman %s %s' % (PATH, PMAN_URL,str(JID))
+        print(cmd)
+        command = open_subprocess(cmd)
+        output = command.stdout.read()                                                     
+        output = str(output, "utf-8")
+        if 'status": true' in output:
+            success_pman_run = success_pman_run + 1
+            num = 0
+            while num<20:
+                cmd = 'bash %s/scripts/run_pman_status %s %s' % (PATH, PMAN_URL,JID)
+                print(cmd)
+                command = open_subprocess(cmd)
+                output = command.stdout.read()                                                     
+                output = str(output, "utf-8")
+                print(output)
+                if '"status": finished' in output:
+                    break
+                else:
+                    time.sleep(int(WAIT))
+                    num = num + 1
+            if '"status": finished' in output:
+                success_pman_status = success_pman_status + 1   
+                cmd = 'bash %s/scripts/run_pfioh_pull %s %s %s' % (PATH, PFIOH_URL ,JID, SIZE)
+                command = open_subprocess(cmd)
+                print(cmd)  	
+                output = command.stdout.read()                                                     
+                output = str(output, "utf-8")
+                print(output)
+                if 'status": true' in output:
+                    success_pfioh_pull = success_pfioh_pull + 1
+                    job_delete()
+                else:
+                    print("Error in Pfioh pull")
+                    print(output)
+        
+            else: 
+                print("Error in obtaining a complete status of Pman's job")
+                job_delete()
+                print(output)
+        else:
+            print("Error in running job in Pman")
+            print(output)
+    else:
+        print("Error in Pfioh push")
+        print(output)
+
+success_pfioh_push = int((success_pfioh_push/RANGE)*100)
+success_pman_run = int((success_pman_run/RANGE)*100)
+success_pman_status = int((success_pman_status/RANGE)*100)
+success_pfioh_pull = int((success_pfioh_pull/RANGE)*100)
+threshold = int(THRESHOLD) 
+
+state = True
+msg = ""
+
+
+if threshold > success_pfioh_push:
+    msg = ", Pfioh Push"
+    state = False
+if threshold > success_pman_run:
+    msg += ", Pman Run"
+    state = False
+if threshold > success_pman_status:
+    msg += ", Pman Status"
+    state = False
+if threshold > success_pfioh_pull:
+    msg += ", Pfioh Pull"
+
+if state==False:
+    msg = msg[2:]
+    with open("export.sh", "w+") as file:
+        file.write('export error="' + msg + '"')
+
```

Comment:
What's the reason behind exporting errors in a shell file?

Reply:
It would be used by CI to send specific error messages to email server. Only way to do that is via environment variables